### ISSUE TITLE
Make Collection Level Access restrictions badges clickable; and open the Access and Use Tab

### ISSUE THREAD

**Issue by ccleeton** [pulibrary/pulfalight#1368]
Users consistently try to click on badges thinking they are buttons.
For example when it shows Access Restrictions,, jump to the access page or the conditions governing access

## Acceptance Criteria

- [x] When going to https://findingaids.princeton.edu/catalog/MC247 or any collection page, clicking the "Restrictions May Apply" button will redirect them to https://findingaids.princeton.edu/catalog/MC247#access (or the appropriate access page for that collection)
- [x] The restrictions badge within search results is not a link. 
- [x] The restrictions badge within the Grouped by Collection is not a link.
- [x] The restrictions badge at the component level is not a link.
- [x] The restrictions badge at the series level is not a link. 

## First Step

Write a test.

**Comment by ccleeton** [comment]
Example: https://findingaids.princeton.edu/catalog/MC247
At the top of the finding aid under the title and the online access green bar, make the "Restrictions may apply. See Access Note" clickable and open the Access & Use tab.

**Comment by ccleeton** [comment]
Users consistently try to click the badge at the top of the finding aid. Users also consistently do not understand if they can actually view the material in person.

**Comment by ccleeton** [comment]
I saw the ticket was closed but I am not seeing it active in the finding aid. Am I missing something?

**Comment by hackartisan** [comment]
@ccleeton I've just deployed; it should be in place now!

**Comment by faithc** [comment]
Hi @hackartisan and @Seanwarren77 -

I noticed that badges at lower levels of description are linking to the collection-level Conditions Governing Access note in the Access + Use tab. If it's possible to implement, badges associated with components should link to the Conditions Governing Access note applied at that component level.

For example, the badge at the [series level here](https://findingaids.princeton.edu/catalog/C1491_c1?onlineToggle=false) shouldn't link to the collection-level note, but rather to the note associated with that component. 

The [badge here](https://findingaids.princeton.edu/catalog/C1491_c00772-80460?onlineToggle=false) should link to the Conditions Governing Access note associated with the component, i.e., "Restricted for 75 years (until 2094)."

Please let Christa or I know if you have any questions.

**Comment by Seanwarren77** [comment]
@faithc good catch thank you for noticing this. We are looking at this now and realizing that this ticket only asked for the Collection level badge to be a link. We are thinking a good next step would be to make the badge not a link at the series and component levels, The same way we just did for the search results. Then we can move on to #1370 and see how it is all coming together. Does that sound good?

**Comment by faithc** [comment]
@Seanwarren77 Thanks for your note and for the reminder that this ticket was specific to collection-level badges/notes! I think we were confused/forgot about the original intent since the clickable functionality had been added at the series level as well. 

I think the way forward you're proposing for how to manage collection-level notes vs. notes at lower levels that relate to this ticket and #1370 makes sense. I wonder if it would be helpful, though, for us to hop on a zoom call just to make sure we're all on the same page given that y'all are working on multiple feature enhancements re: access notes and it's been several months since we created some of these tickets.
